### DISCUSSION TITLE
Is the dragonfly compatible with redisson memory grid?

### DISCUSSION THREAD

**Discussion by sniper-xx** [dragonflydb/dragonfly#1138] [Q&A]
**Did you search GitHub Issues and GitHub Discussions First?**
Many users may find their feature is already being discussed. Help us keep duplicates to a minimum by searching for your feature first to see if it is already in progress.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by romange** [comment]
yes, it's compatible
